File: BlotoutAnalytics/Models/Item.m
//
//  Item.m
//  BlotoutAnalyticsSDK
//
//  Created by Nitin Choudhary on 28/11/21.
//  Copyright © 2021 Blotout. All rights reserved.
//

#import "Item.h"

@implementation Item

-(instancetype)init
{
    self = [super init];
    self.item_sku = @"";
    self.item_name = @"";
    self.item_price = [NSNumber numberWithInt:0];
    self.item_category = @[];
    self.item_currency = @"";
    self.item_quantity = [NSNumber numberWithInt:0];
    
    return self;
}

@end

File: BlotoutAnalytics/Models/Item.swift
//
//  Item.swift
//  BlotoutAnalyticsSDK
//
//  Created by Poonam Tiwari on 21/03/22.
//

import Foundation

public class Item: NSObject {
    
    public var item_id = "" //Required Parameter
    public var item_name: String?
    public var item_sku: String?
    public var item_category: [AnyHashable]?
    public var item_price: NSNumber?
    public var item_currency: String?
    public var item_quantity: NSNumber?
}
